include/fmt/chrono.h

```diff
@@ -912,7 +912,7 @@ template <typename Derived> struct null_chrono_spec_handler {
 };
 
 struct tm_format_checker : null_chrono_spec_handler<tm_format_checker> {
-  FMT_NORETURN void unsupported() { FMT_THROW(format_error("no format")); }
+  FMT_NORETURN FMT_INLINE void unsupported() { FMT_THROW(format_error("no format")); }
```

Comment:
Here and in other places in the `detail` namespace - `inline` is not needed since it is not exported from the module.

Reply:
As above, `detail` namespace functions will still generate linker errors if neither `inline` nor dll-exported.